acceptance/help_test.go

```diff
@@ -45,6 +46,7 @@ Usage: om [options] configure-authentication [<args>]
   -u, --username             string  admin username for the Ops Manager VM (not required for unauthenticated commands)
   -p, --password             string  admin password for the Ops Manager VM (not required for unauthenticated commands)
   -k, --skip-ssl-validation  bool    skip ssl certificate validation during http requests (default: false)
+  -r, --request-timeout      int     timeout in seconds for HTTP requests to Ops Manager (default: 1800)
```

Comment:
This default isn't actually enforced anywhere. Please add a test and check the default.